docs/snippets/github-secret-store.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: SecretStore
+metadata:
+  name: github-secret-stor
+  namespace: external-secrets
+spec:
+  provider:
+    github:
+      appID: "0000000"
+      installID: "00000000"
```

Comment:
Could you please add a page to document the GitHub provider?
Please point to the appropriate documentation pages on GitHub on how to set up github apps and how to acquire authentication credentials.